kernel_gateway/tests/test_gatewayapp.py

```diff
@@ -67,6 +68,7 @@ def test_config_env_vars(self):
         self.assertEqual(app.keyfile, '/test/fake.key')
         self.assertEqual(app.certfile, '/test/fake.crt')
         self.assertEqual(app.client_ca, '/test/fake_ca.crt')
+        self.assertEqual(app.ssl_version, 5)
         self.assertEqual(app.trust_xheaders, False)
 
     def test_trust_xheaders(self):
```

Comment:
Since the defaulting of `ssl_options` will be deferred, it might be helpful to have a test that ensures the default of `5` is being used when no options other than one of the cert options are specified.